src/integration/pages/storyPage/canonicalIncludeTests.js

```diff
@@ -3,7 +3,7 @@ export default () => {
     describe('IDT2', () => {
       it('I can see a "dataPic"', () => {
         const scriptEl = document.querySelector(
-          'script[src="https://news.test.files.bbci.co.uk/include/idt2/static/js/dataPic.8f264bc5.js"]',
+          'script[src*="https://news.test.files.bbci.co.uk/include/idt2/static/js/dataPic"]',
         );
         expect(scriptEl).toBeInTheDocument();
         expect(scriptEl).toMatchSnapshot();
```

Comment:
Tests are failing due to the snapshot not matching. Should we remove the snapshot test? We don't want to have to keep updating these tests if the datapic include is updated / republished.

Reply:
Sounds like a good plan.